layouts/default.vue

```diff
@@ -99,7 +108,19 @@
     },
 
     watch: {
-      '$i18n.locale': '$fetch'
+      '$i18n.locale': '$fetch',
```

Comment:
At present the announcement is always in English.

Let's make use of the fact that we are already here watching for locale changes, and in the `fetch` function, set the announcement text based on the user's locale, using [`$announcer.set`](https://vue-announcer-v2.surge.sh/guide/announcer.html#methods) on the VM, with messages from `$i18n`.

Reply:
The Vue Announcer also offers the [$announcer.setComplementRoute](https://vue-announcer-v2.surge.sh/guide/announcer-router.html#announcer-setcomplementroute-complementroute) method specifically for this case so I'm using that. 
complementRoute refers to the string that is appended to the page title in the announcements after route changes.

Reply:
And I also call it on the mount method to have the translated value always. Otherwise it would only check for translations after the user changes de language.